best-practices.md

```diff
@@ -202,12 +288,72 @@ register the media type with IANA, so that other STAC clients can find it easily
 [acceptable to not register](https://stackoverflow.com/questions/29121241/custom-content-type-is-registering-with-iana-mandatory) 
 it. It is relatively easy to [register](https://www.iana.org/form/media-types) a `vnd` media type.
 
-## Static and Dynamic Catalogs
+### Asset Roles
+
+[Asset roles](item-spec/item-spec.md#asset-roles) are used to describe what each asset is used for. They are particular useful 
+when several assets have the same media type, such as when an Item has a multispectral analytic asset, a 3-band full resolution 
+visual asset, a downsampled overview, and an cloud mask, all stored as Cloud Optimized GeoTIFF's. It is recommended to use a role
+for every asset available, and just pick a sensible name for the role if there is not a clear one in the [Asset Role 
+Types](item-spec/item-spec.md#asset-role-types). You can also propose new roles to the specification by raising an issue/PR. 
+
+In addition to the thumbnail, data and overview [roles listed](item-spec/item-spec.md#asset-role-types) in the Item spec, there
+are a number of roles that are emerging in practice, but don't have enough widespread use to justify standardizing them. So if
+you want to re-use other roles then try to find them on the list below, and also feel free to suggest more to include here.
+
```

Comment:
Maybe also mention their use in Extensions -- for example, we use (proprietarily) a role of `labels` on assets in Items using the Label Extension, and there's discussion of changing that spec to use this role instead of requiring the asset be named 'labels'

Reply:
Sounds good. Will add `labels` to the list, and mention extensions.

Reply:
Actually I don't think I'll add this yet, since right now it's not actually specified in the label extension. When #1001 is complete we can add it (or it can add it to this list if we merge this one first).